packages/image_picker/image_picker_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.8.6+7
+
+* Improves Bitmap resizing.
```

Comment:
Could you make this a little more specific? I wouldn't know what improvement is being made.